### ISSUE TITLE
[Bug]: INSERT null failed when all columns have default values

### ISSUE THREAD

**Issue by domingozhang** [matrixorigin/matrixone#1155]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):b84b704a51f85f9bc186bdfbfe2a86576414d6c2
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

CREATE TABLE t_default (
i INT DEFAULT -1,
v VARCHAR(10) DEFAULT 'abc',
c CHAR(10) DEFAULT '',
price DOUBLE(16,2) DEFAULT 0.00
);

INSERT INTO t_default () VALUES();
ERROR 1105 (HY000): unknown error:[22000]constant value out of range

INSERT INTO t_default () VALUES(DEFAULT, DEFAULT, DEFAULT, DEFAULT);
ERROR 1105 (HY000): unknown error:[22000]constant value out of range

### Expected Behavior

Query OK, 1 row affected (0.00 sec)

**Comment by m-schen** [comment]
it is due to default value `-1` build in a wrong way.
1 is no problem.
will fix it at 0.2

**Comment by m-schen** [comment]
has been fix at factor branch. 
And you can verify this issue at factor branch or after factor branch has merged into main factor.
